app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -187,7 +196,10 @@ def get_optimal_flavor_for_tenant(tmpl, t)
           best_mem = Float::INFINITY
           opt_fl = nil
           mfs.each do |f|
-            present_cost = tmpl.get_hourly_cost_for(f)
+            fofs = f.flavor_os_families
+            osf = fofs.detect { |fof| fof.os_family_id ==
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.